pages.es/common/autoflake.md

```diff
@@ -0,0 +1,20 @@
+# autoflake
+
+> Una herramienta para eliminar importaciones y variables no utilizadas del código Python.
+> Más información: <https://github.com/myint/autoflake>.
+
+- Elimina las variables no utilizadas de un único archivo y muestra la diferencia:
+
+`autoflake --remove-unused-variables {{archivo.py}}`
+
+- Elimina las importaciones no utilizadas de varios archivos y muestra las diferencias:
+
+`autoflake --remove-all-unused-imports {{archivo1.py}} {{archivo2.py}} {{archivo3.py}}`
```

Comment:
```suggestion
`autoflake --remove-all-unused-imports {{ruta\al\archivo1.py}} {{ruta\al\archivo2.py}} {{ruta\al\archivo3.py}}`
```

Reply:
Good suggestion, I will update the same in the English page.

Reply:
Updated your current suggestion to tldr syntax.